blockchain/chainio.go

```diff
@@ -1080,6 +1080,32 @@ func SerializeUtreexoRoots(numLeaves uint64, roots []utreexo.Hash) ([]byte, erro
 	return w.Bytes(), nil
 }
 
+// SerializeUtreexoRootsHash serializes the numLeaves and the roots into a byte slice.
```

Comment:
> We don't need this because we can just use
> 
> https://github.com/utreexo/utreexod/blob/e166f1004e45c98c17d26ec7bad786519fd544a3/blockchain/chainio.go#L1060
> 
> I don't think there's any code where utreexo roots are fetched as `chainhash.Hash` instead of `utreexo.Hash` anyways. If there needs to be the conversion, you can do a type cast like so:
> 
> `chainHashRoot := chainhash.Hash(root)`

Ahh yes.. I saw that function. I had to create a different function, because methods like `idx.chain.FetchUtreexoViewpoint(blockHash)`  (idx is of type `UtreexoProofIndex`) returns a `UtreexoViewPoint`, and calling the `viewPoint.GetRoots()` returns results as a `chainhash.Hash` object. `idx.FetchUtreexoState(dbtx, blockHash)` also returns roots as a `chainhash.Hash`.
So the conversion you highlighted above, returns a `chainhash.Hash`, whereas the original `func SerializeUtreexoRoots(numLeaves uint64, roots []utreexo.Hash) ([]byte, error) { ` requires a `utreexo.Hash`